includes/download.yaml

```diff
@@ -38,6 +38,7 @@
         remote_src: true
       register: gh_runner_path_unarchived
       become: true
+      ignore_errors: "{{ ansible_check_mode }}"
```

Comment:
with https://github.com/ansible/ansible/issues/56345 in mind, I would honestly advise against implementing this `ignore_errors`.

The check works when you do a dry run, but would eventually ignore errors even when not dry running.

Reply:
TIL! I'll revert that change 👍